### DISCUSSION TITLE
Daily 11/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#23] [Daily Sprint]
* Que hice ayer?
* Que haré hoy?
* Problemas o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer subí los apuntes de la weekly del día domingo. También subí, a la wiki, la base de la documentación falta hacerle correcciones, agregar historias de usos y los datos de los participantes.
Hoy  instalaré el nodejs y el tsc, continuaré con la corrección de la documentación y tengo pendiente hacer el exámen nivelatorio de inglés.
El impedimento que puedo tener es la falta de tiempo por otras tareas que debo realizar

**Comment by BraianTroncoso** [comment]
Ayer me puse al día con algunos temas del ISPC y repasé temas de backend sobre Python, Java, Js. 
Hoy voy a empezar leer la documentación del ISPC sobre POO con BDD.
El impedimento que puedo tener es la falta de tiempo por otras tareas que debo realizar por cuestiones laborales.

**Comment by rpgrca** [comment]
Ayer estuve armando el script de sql en base al viejo, voy a estar modificándolo para ajustar las nuevas tablas que tiene lo nuevo. Todavía hay que hacerles algunos cambios pero trato de trabajar alrededor. Estuve leyendo lo de Angular, lo de POO más o menos ya lo conozco todo. A la tarde tengo clases de programación y me asignaron un proyecto en el trabajo así que voy a estar con un poco menos de tiempo por acá.

**Comment by Tarias89** [comment]
Ayer me conecte a la clase de mi comisión, al finalizar me dedique a ponerme al día con el material cargado.
Hoy voy a continuar con el material de estudio e interiorizarme en el proyecto.
Mi impedimento es la falta de tiempo, por tareas laborales y el cuidado de mi hijo.

**Comment by AhumadaCeleste** [comment]
Ayer finalice con el modelo entidad de relacion. Tuve dificulta para generar el script. Hoy lo resolví, ahora lo agrego. No funciónaba el vinculo para subir el examen nivelatorio de inglés. Pude hacerlo hoy.  Mañana  en lo que resta del día quiero avanzar con los diagramas de uso y el diseño de pantallas de logüin etc.

**Comment by Brian-Almada** [comment]
Estos días fueron perdidos por cuestiones de salud. Recién hoy comencé con las tareas de ingles y ahora me voy a poner al día con revisar las issues y todo lo que está pasando en el proyecto.

**Comment by KAAcevedo** [comment]
Asistí a una reunión donde se habló en términos generales de todo el proyecto. 
Revisé el repositorio, y cargué mis datos en la Wiki y en el formulario IEEE830
